### ISSUE TITLE
Route not showing

### ISSUE THREAD

**Issue by NathanFumerola** [Funbit/ets2-telemetry-server#147]
I have a gps one for this and when taking a job it doesnt show the route in ats

**Comment by KS-HTK** [comment]
Hey, if you are using [mike-koch/ets2-mobile-route-advisor](https://github.com/mike-koch/ets2-mobile-route-advisor) this is not an issue. It is a limitation of the mra(mobile-route-advisor).
While the telemetry server gives data about current location and job, it does not supply any information about the route the game tells you to take.

Furthermore the mra only places an arrow and loads the map at that location, it is not capable of drawing a route onto the map. The map is based on tiles (small image files).
For more information about raster tiles and vector tiles [this article](https://bachasoftware.com/what-is-tile-and-differentiate-between-raster-tile-and-vector-tile/) gives a good overview.
As I understand maps and mapping, raster data maps do not give information about the route, all tiles are per calculated images and are shown based on a xy location in the raster.
To be able to draw the route the map itself would need to be either vector data tiles or a map service. Those could then be used by frameworks like leaflet to display the map an a route.

If you want to take the time to work on this issue, I would suggest starting with a way of getting vector tiles of the game map. [dariowouters/ts-map](https://github.com/dariowouters/ts-map) seems to be a software capable of generating raster tile maps of the game by extracting the information out of the game files. Maybe that software could be expanded so it is capable of generating vector tile maps.

TL;DR: This is not a issue of the ets2-telemetry-server and it is not a issue of the moblie-route-advisor, rather it is a limitation of the way the mra is implemented due to not having access to the vector data of the game map.

**Comment by IgazHarcos** [comment]
It never showed. The necessary data is not in the API to do this.
